backend/src/main/java/ai/giskard/web/dto/ExplainTextResponseDTO.java

```diff
@@ -0,0 +1,22 @@
+package ai.giskard.web.dto;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.dataiku.j2ts.annotations.UIModel;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@UIModel
+public class ExplainTextResponseDTO {
+    List<String> words = new ArrayList<String>();
```

Comment:
you can skip type in the constructor if it's already specified in the variable type
```suggestion
    List<String> words = new ArrayList<>();
```